docs/zh/latest/external-plugin.md

```diff
@@ -0,0 +1,104 @@
+---
+title: 外部插件
+---
+
+<!--
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+-->
+
+## 什么是外部插件和插件运行程序
+
+APISIX 支持使用 Lua 语言写插件，这种类型的插件在 APISIX 内部执行。
+有时候你想使用其他语言来开发插件，因此，APISIX 支持以边车的方式加载和运行你写的插件。
+这里的边车就叫做插件运行程序，你写的插件叫做外部插件。
```

Comment:
We should keep the terms unchanged:
- External Plugin
- Plugin Runner
- Sidecar

Reply:
okay~ done~ thanks~ @spacewander